pkg/scheduler/core/generic_scheduler.go

```diff
@@ -208,6 +201,13 @@ func (g *genericScheduler) Schedule(ctx context.Context, prof *profile.Profile,
 		}, nil
 	}
 
+	// Run "prescore" plugins.
```

Comment:
perhaps we should move it inside ```prioritizeNodes```?

Reply:
+1

Reply:
Done. I did that for PreFilter as well. The only counter argument is that we loose the tracing step, but that is not necessarily a big deal.